### ISSUE TITLE
[BUG] - Gear up on loading screen (loaded at a gate) and inside the hangar

### ISSUE THREAD

**Issue by KJA1582** [invernyx/md11-bugs#1644]
### Describe the bug

Nose and main gear are shown in up position when viewing the plane inside the hangar and loading at a gate

### Expected behaviour

Reopen of #1207

> The gear should be down both inside the hangar and when loading at a gate. (loading directly on a runway is fine)
> Note: The gear appears in up position only before hitting 'start the flight', once inside the cockpit it is shown properly
> 
> This is valid for all versions (passenger and cargo with both engine variants)
> and also happens if you take thumbnail pictures via SDK Hangar camera

This can be fixed by setting the LVARs that drive the animation inside the startup .FLT files.
The ones mostly affected would be hangar.FLT and apron.FLT.

Files: https://github.com/invernyx/md11-bugs/issues/1644#issuecomment-2234051214

Section in question:
```
[LocalVars.0]
MD11_EXT_LEFT_GEAR=100
MD11_EXT_RIGHT_GEAR=100
MD11_EXT_CTR_GEAR=100
MD11_EXT_NOSE_GEAR=100
MD11_EXT_CHOCKS=1
MD11_EXT_R_OUTB_AIL=100
MD11_EXT_STAB_TRIM=24.242424242424242209
MD11_EXT_LOWER_RUD=45.322704454879321645
MD11_EXT_R_INB_AIL=100
MD11_EXT_UPPER_RUD=45.322704454879321645
MD11_YOKE_LEFTRIGHT=49.999084433539032091
MD11_YOKE_FOREAFT=50.000915566460967909
MD11_TILLER=50
MD11_OPT_ANALOG_STBY=1
MD11_OPT_PIP_PYLON=0
MD11_AOVHD_ENG1FIRE_KB=1
MD11_AOVHD_ENG3FIRE_KB=1
MD11_AOVHD_ENG2FIRE_KB=1
md11_ext_non_er_tail=1
```

### Evidence

![image](https://github.com/user-attachments/assets/e0e93f2f-122b-4231-a724-822a29fe2423)
![image](https://github.com/user-attachments/assets/7eb3a522-bf72-4b4c-8286-6ba248d2b35c)

### Simulator

Microsoft Flight Simulator 2020 (MSFS)

### Aircraft Type

Freighter GE

### Agreement

- [X] Yes

**Comment by KJA1582** [comment]
I've adjusted some more so that the hangar cockpit view doesn't have a sideways yoke

[FLT files.zip](https://github.com/user-attachments/files/16273677/FLT.files.zip)

**Comment by Jmknights93** [comment]
> I've adjusted some more so that the hangar cockpit view doesn't have a sideways yoke
> 
> [FLT files.zip](https://github.com/user-attachments/files/16273677/FLT.files.zip)

Are these files ok to use in both "TFDi_Design_MD-11_GE" & "TFDi_Design_MD-11F_GE" folders?

**Comment by KJA1582** [comment]
> > I've adjusted some more so that the hangar cockpit view doesn't have a sideways yoke
> > [FLT files.zip](https://github.com/user-attachments/files/16273677/FLT.files.zip)
> 
> Are these files ok to use in both "TFDi_Design_MD-11_GE" & "TFDi_Design_MD-11F_GE" folders?

Wouldn’t do it, though I can’t see why not.
The safer option would be to edit the Pax files by hand and copy in the LocalVar block.

**Comment by Jmknights93** [comment]
> > > I've adjusted some more so that the hangar cockpit view doesn't have a sideways yoke
> > > [FLT files.zip](https://github.com/user-attachments/files/16273677/FLT.files.zip)
> > 
> > 
> > Are these files ok to use in both "TFDi_Design_MD-11_GE" & "TFDi_Design_MD-11F_GE" folders?
> 
> Wouldn’t do it, though I can’t see why not. The safer option would be to edit the Pax files by hand and copy in the LocalVar block.

Yeah, that might be the better option. Thanks.

**Comment by turbofandude** [comment]
Thank you all! This has been resolved in 1.0.8+
